app/views/fields/paperclip/_index.html.erb

```diff
@@ -1 +1,5 @@
+<% if field.url_only? -%>
```

Comment:
It's a habit of mine with control statements. It chomps the newline that appears after the closing tag which stops it appearing in the output. Doesn't really matter with HTML but I just see it as slightly cleaner.